formats/json/jvmMain/src/kotlinx/serialization/json/schemaCache.kt

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright 2017-2021 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.serialization.json
```

Comment:
Please follow the naming conventions of files.

I suppose current ones are auto-generated (and it's really annoying)